72NetworkBootstraped/Controllers/AccountController.cs

```diff
@@ -126,7 +126,7 @@ public ActionResult SignUp(SignUpModel model)
           ModelState.AddModelError("", ErrorCodeToString(e.StatusCode));
         }
       }
-
+      ViewBag.Countries = new SelectList(Util.ListOfCountries());

```

Comment:
When there is validation error in the form being posted back, ViewBag.Countries is empty and the View() encounteres error while being rendered.